gradle.properties

```diff
@@ -15,7 +15,7 @@ developers=Graeme Rocher
 githubCoreBranch=3.4.x
 bomProperty=micronautJaxrsVersion
 
-jaxrsapi=https://jax-rs.github.io/apidocs/2.0
```

Comment:
I think the new repo is https://github.com/jakartaee/rest and javadoc should be jaxrsapi=https://jakartaee.github.io/rest/apidocs/2.1.6/